protocols/doodles/index.json

```diff
@@ -0,0 +1,8 @@
+{
+  "cname": "doodles",
+  "description": "Doodles is a community-driven collectibles NFT project. Doodle NFTs are made up with over a hundred exciting traits of faces, hair, hats, body and backgrounds. Each Doodle is a unique, non-fungible token (NFT) on the Ethereum blockchain.",
+  "path": "doodles",
+  "isEnabled": true,
+  "type": "snapshot",
+  "safeAddress": "0xDcd382bE6cC4f1971C667ffDa85C7a287605afe4"
```

Comment:
Hey, can you change this to `treasuryAddress: "0xDcd382bE6cC4f1971C667ffDa85C7a287605afe4"`?

Reply:
@Ondkloss

Reply:
@sudheerDev @Rashmi-278 is `treasuryAddress` a new format? Just to understand. I was working from:
https://docs.boardroom.info/adding-your-project/adding-your-project/protocol-information

Which only mentions `safeAddress`.

Reply:
@Ondkloss  Yes, we will soon update the docs to reflect that change.